<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ZeeshanAhmed" failures="1" tests="5" name="fiTest" time="5.491" errors="0" timestamp="2022-03-20T12:43:09 PKT">
  <testcase classname="com.uiafw.cn.pn.testscripts.FirstTest" name="testLoginF" time="0.005">
    <skipped/>
  </testcase> <!-- testLoginF -->
  <testcase classname="com.uiafw.cn.pn.testscripts.FirstTest" name="testLoginF" time="0.016">
    <skipped/>
  </testcase> <!-- testLoginF -->
  <testcase classname="com.uiafw.cn.pn.testscripts.FirstTest" name="testLoginF" time="0.004">
    <skipped/>
  </testcase> <!-- testLoginF -->
  <testcase classname="com.uiafw.cn.pn.testscripts.FirstTest" name="testLoginF" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.uiafw.cn.pn.helper.assertion.AssertionHelper.makeFalse(AssertionHelper.java:33)
at com.uiafw.cn.pn.testscripts.FirstTest.testLoginF(FirstTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testLoginF -->
  <testcase classname="com.uiafw.cn.pn.testscripts.FirstTest" name="testLoginP" time="0.001"/>
</testsuite> <!-- fiTest -->
